sdk/docs/core/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Release History
 
 
+## 1.0.0-preview.4 (Unreleased)
+
+
 ## 1.0.0-preview.3 (Unreleased)
```

Comment:
Remove `unreleased` and add some changelog text?

From core, there were a bunch of bug fixes, and JSON stack API changes and rename (json parser -> json reader, json builder -> json writer).

```suggestion
## 1.0.0-preview.3 (2020-07-20)
```

Reply:
Updating the change log is supposed to happen before the release. This PR is opened automatically for the purpose of incrementing the version and adding a new entry into CHANGELOG.md for the next release.

Reply:
Should adding the date to the previous, unreleased, release also be part of the auto-update?